frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -133,6 +141,24 @@ export const LogDetails = ({ row, queryTerms }: Props) => {
 					/>
 				</Box>
 			)}
+			{source && (
```

Comment:
Heads up @ccschmitz - this will conflict with #4711. I'll leave a suggestion on #4711 to how to dedupe some logic.